src/common/snippets/src/op/broadcastload.cpp

```diff
@@ -29,5 +29,10 @@ std::shared_ptr<Node> snippets::op::BroadcastLoad::clone_with_new_inputs(const O
 }
 
 void snippets::op::BroadcastLoad::validate_and_infer_types() {
+    // BroadcastLoad has memory access port only on output
```

Comment:
"only on input"? Otherwise it contradicts the assert on L36

Reply:
On input, sure. Thanks!